python/mlcroissant/mlcroissant/_src/core/rdf.py

```diff
@@ -25,6 +25,9 @@ def make_context(ctx=None, **kwargs):
     context = {
         "@language": "en",
         "@vocab": "https://schema.org/",
+        "arrayShape": (
+            "cr:arrayShape" if ctx is not None and ctx.is_latest_version() else None
```

Comment:
The condition should be `ctx.version >= 1.1`? Otherwise, when increasing the version, we'll lose cr:arrayShape?

Same below.

Reply:
Addressed in https://github.com/mlcommons/croissant/pull/807/commits/369c07f5cd31fd2b64198b7fed4cf81502b007a0